testing/deployer/Makefile

```diff
@@ -0,0 +1,84 @@
+SHELL := /bin/bash
```

Comment:
I restored this makefile as it was removed during the submoduleification.

Mostly this was for the `make update-defaults` target which helps with bumping the versions of the containers.